aggregator_core/src/datastore/test_util.rs

```diff
@@ -57,8 +57,7 @@ impl EphemeralDatabase {
             let shutdown_barrier = Arc::clone(&shutdown_barrier);
             move || {
                 // Start an instance of Postgres running in a container.
-                let container_client = testcontainers::clients::Cli::default();
-                let db_container = container_client.run(RunnableImage::from(Postgres::default()));
+                let db_container = RunnableImage::from(Postgres::default()).start();
```

Comment:
If the reason we're still using the sync interface in this one place is because of the use of a (synchronous) thread, I think this would be pretty easy to replace with an async task -- other than the testcontainers interface & the async channel (which are already async), we use a `Barrier` to communicate shutdown. This Barrier could pretty easily be replaced by another `oneshot` channel, for example, to allow us to async-ify all of the blocking operations.